src/controllers/api/users.js

```diff
@@ -0,0 +1,93 @@
+// const connectToDatabase = require("../../../config/connection");
+const { User } = require("../../models");
+const getAllUsers = async (req, res) => {
+  try {
+    // await connectToDatabase();
+    const allUsers = await User.find({});
+    return res.json({ success: true, allUsers });
+  } catch (error) {
+    console.log(`[ERROR]: Failed to get users | ${error.message}`);
+
+    return res
+      .status(500)
+      .json({ success: false, error: "Failed to get users" });
+  }
+};
+
+const getUserById = async (req, res) => {
+  try {
+    const { id } = req.params;
+    const userById = await User.findById(id);
+
+    if (!userById) {
+      console.log(`[ERROR]: User not found`);
+      return res.status(404).json({ success: false, error: "Users not found" });
+    }
+
+    return res.json({ success: true, userById });
+  } catch (error) {
+    console.log(`[ERROR]: Failed to get user | ${error.message}`);
+    return res
+      .status(500)
+      .json({ success: false, error: "Failed to get user" });
+  }
+};
+
+const createUser = async (req, res) => {
```

Comment:
Remember to leave spacing within functions. You have quite a few functions within your controllers that are big blocks of code and this makes it quite difficult to read.